modules/ts/misc/run.py

```diff
@@ -879,7 +918,7 @@ def getRunArgs(args):
     run_args = getRunArgs(args[1:] or ['.'])
 
     if len(run_args) == 0:
-        print >> sys.stderr, "Usage:", os.path.basename(sys.argv[0]), "[options] [build_path]"
+        print >> sys.stderr, "Usage:", os.path.basename(sys.argv[0]), "[options] build_path"
```

Comment:
@alalek Updated this part and OptionParser message with details concerning of the build_path